pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -34,6 +35,10 @@
 sys.path.insert(0, os.path.join(ROOT.parent.parent.parent.parent.parent, "maintainers", "scripts"))
 import pluginupdate
 
+RUST_PLUGINS = [
+    "vim-clap"
+]
```

Comment:
I think this is the most annoying part, if someone has a better idea of a way to determine which plugins to use please let me know.

If people agree though that it's good enough I will update the Vim README in this PR to mention this.

Reply:
I see there's some Nix magic in `GET_PLUGINS` I might be able to use...